.circleci/config.yml

```diff
@@ -2,6 +2,8 @@ version: 2.1
 
 orbs:
   node: circleci/node@4.2.1
+  terraform: circleci/terraform@2.1.0
+  jq: circleci/jq@2.2.0
```

Comment:
We leveraged this [Terraform Orb](https://circleci.com/developer/orbs/orb/circleci/terraform) for CircleCI to more easily install Terraform and use its built in jobs, and this [JQ orb](https://circleci.com/developer/orbs/orb/circleci/jq) which we use to parse JSON responses when setting environment variables for Terraform below.

Reply:
As an additional note, these are both certified and published by CircleCI so they should not cause any runtime issues on the HHS Circle CI instance.